static/js/components/NewApplication_test.js

```diff
@@ -61,6 +60,21 @@ describe("NewApplication", () => {
     )
   })
 
+  it("shows a message if no bootcamps are available for application", async () => {
+    helper.handleRequestStub.withArgs(getBootcampsUrl).returns({
+      status: 200,
+      body:   []
+    })
+
+    const { wrapper } = await renderPage()
+    assert.isFalse(wrapper.find("Dropdown").exists())
+    const message = wrapper.find("p")
+    assert.equal(
+      message.text(),
+      "There are no bootcamps currently available for application."
```

Comment:
Minor nitpick: The "for application" end of the sentence sounds a bit awkward to me, might be better to cut that out.  I'm going to approve this anyway since that's a highly subjective opinion.